extensions/sun-moon-times/src/components/CityListView.tsx

```diff
@@ -0,0 +1,51 @@
+import { List } from "@raycast/api"
+import { useFetch } from "@raycast/utils"
+import { useState } from "react"
+import { ressourceUrl } from "../../assets/ressourceUrl"
+import { ResponseData } from "../../types/ResponseData"
+import { CityListItemView } from "./CityListItemView"
+import { useFavorites } from "./FavoritesProvider"
+
+export const CityListView = () => {
+    const [searchText, setSearchText] = useState<string>("")
+    const { isLoading, data } = useFetch<ResponseData>(ressourceUrl(searchText))
+
+    const { favorites } = useFavorites()
+
+    return (
+        <List isLoading={isLoading} onSearchTextChange={setSearchText}>
```

Comment:
search is kind of jumpy – what about using `throttle`?
```suggestion
        <List isLoading={isLoading} onSearchTextChange={setSearchText} trottle>
```